### PR TITLE
New blog post re supported modules changes

### PR DESCRIPTION
**Pull request by Aaron-Rob**

### CODE REVIEW

File: content/blog/posts/2022-06-30-changes-to-supported-modules.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+# Changes to Supported Modules
+
+We are continually reviewing our Supported modules list and understanding usage and value to users. We want to ensure that the CAT team 
+is prioritizing support for the modules that matter the most to you.
+
+We are taking the step to remove Puppet support from the below listed models, this change will happen effective 15 July 2022. While these 
+will not be supported any longer by Puppet, the repos will stay active for use and community contributions, as such they are not going away.
+
+* [Helm](https://forge.puppet.com/modules/puppetlabs/helm)
```
Comment (david22swan) [inline content/blog/posts/2022-06-30-changes-to-supported-modules.md:15]: One thought, but we should probably do a release of all of these prior to removing support, just to be nice.

Comment (chelnak) [reply content/blog/posts/2022-06-30-changes-to-supported-modules.md:14]: Seems sensible!

File: config.toml [hunk 2]
```diff
@@ -219,7 +219,7 @@ enable = false
 #         icon = "fa fa-user-friends"
 #         desc = "Project Maintainers"
 [params.home] # Takes a RelPermalink e.g. "/content-and-tooling-team/blog/posts/return-of-community-day/"
-pinnedPost = "/content-and-tooling-team/blog/posts/return-of-community-day/"
+pinnedPost = ""
```
Comment (chelnak) [inline config.toml:222]: Should we pin this post?

Comment (Aaron-Rob) [reply config.toml:222]: I think we have removed pinning from the blogs

Comment (petergmurphy) [reply config.toml:222]: We could pin it for a bit as it's more important than the next weekly update

**Review by chelnak — APPROVED** [review]

File: config.toml (hunk 2 above)
Comment (petergmurphy) [reply config.toml:222]: Scratch that, no pinned is best

**Pull request merged**